### PR TITLE
fix bug as #88 and add debug msg to locate libbpf package

### LINKED ISSUE #88: a bug case kind cluster always start up
https://github.com/sustainable-computing-io/kepler-action/blob/e32a48b7e3b136f18809e4bc45d297326927a10f/index.js#L103C27-L103C31

as the default value is kind but not null, the kind cluster will always start up...

### PR DESCRIPTION
**Pull request by SamYuan1990**
fix #88 
add debug msg to locate libbpf package

### CODE REVIEW

File: verify.sh [hunk 1]
```diff
@@ -41,6 +41,8 @@ function verify_libbpf() {
         echo "no libbpf package found"
         exit 1
     fi
+    sudo apt-get install -y mlocate
+    locate libbpf
```
Comment (vprashar2929) [inline verify.sh:45]: Won't `dpkg -L libbpf-dev` work?

Comment (sthaha) [inline verify.sh:45]: My preference would be to rely on pkgconfig to check the all things are installed. 
```
pkg-config --modversion libbpf
```
This is already installed and IMHO, should be used to obtain the `GO_CFLAGS` and `--static` library

Comment (rootfs) [reply verify.sh:45]: per [this](https://forums.veeam.com/vmware-vsphere-f24/ubuntu-22-04-mlocate-t81712.html) message, `mlocate` is gone in newer ubuntu

**Comment by rootfs** [comment]
@SamYuan1990 can you rebase? it is ready to merge

**Review by rootfs — APPROVED** [review]

File: verify.sh (hunk 1 above)
Comment (SamYuan1990) [reply verify.sh:48]: https://github.com/sustainable-computing-io/kepler-action/actions/runs/7059814600/job/19218145143?pr=89#step:4:16 @sthaha , could you please try `pkg-config` in another PR?

**Comment by SamYuan1990** [comment]
> @SamYuan1990 can you rebase? it is ready to merge

sorry I am very busy in recently, and I just rebase it. Could you please merge this PR and make a new release after merge?
as this PR contains bug fix to avoid create kind cluster every time.... I hope a new version asap. and Dep Bot will help us bump up in kepler.

**Comment by SamYuan1990** [comment]
or by default I will try to merge this PR before Monday as merge it is... and create a new release accordingly. (well, honestly I don't want to merge myself)

**Comment by rootfs** [comment]
@SamYuan1990 is it a good time to cut the release?

**Comment by SamYuan1990** [comment]
> @SamYuan1990 is it a good time to cut the release?

just because it's a bug fix release.

**Pull request merged**